src/pages/iou/request/step/IOURequestStepScan/index.tsx

```diff
@@ -298,6 +280,7 @@ function IOURequestStepScan({
             </View>
 
             <AttachmentPicker>
+
                 {({openPicker}) => (
```

Comment:
@codinggeek2023 hi, `AttachmentPicker` is not migrated to TS yet, thats why you are getting error here , we agreed that when we are getting errors in TS files because of js files we can add `@ts-expect-error` comment to stop getting this error to time when JS file is migrated so you can put here something like that 
```
{/* @ts-expect-error TODO: Remove this once AttachmentPicker (https://github.com/Expensify/App/issues/25134) is migrated to TypeScript. */}

Reply:
Will check it later or tomorrow morning !